code/modules/mob/language.dm

```diff
@@ -268,6 +268,17 @@
 	flags = RESTRICTED | WHITELISTED
 	syllables = list("click","clack")
 
+/datum/language/moth
+	name = "Gypsikin"
+	desc = "The noise made by rubbing its furry antennae together is actually a complex form of communication for Kidan."
```

Comment:
That is because its way to late for me to be writing code, and I didn't fix all the old text from what this originally was. I'll fix it when I am not about to pass out.